src/all_optimizations.py

```diff
@@ -19,7 +19,7 @@
 plt.style.use(os.path.join('..', 'configs', 'mplstyle.yaml'))
 
 
-def fte(DATA_DIR, start_frame, end_frame, dlc_thresh):
+def fte(DATA_DIR, DLC_DIR, start_frame, end_frame, dlc_thresh, plot: bool = False):
```

Comment:
I don't see the plot variable being used anywhere in the code. If its not being used then perhaps remove?

Reply:
`plot` is used on line 41 to disable showing a matplotlib window for CLI.